ZLIB, JPEG, PNG, OPENSSL 의존성을 추가하며 빌드 로그는 아래와 같습니다.

```
$ python build.py build vncserver -p x64 -c release --vs-ver 17
...
...
...

-- The C compiler identification is MSVC 19.43.34808.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Community/VC/Tools/MSVC/14.43.34808/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found ZLIB: C:/gtk-build/gtk/x64/release/lib/zlib.lib (found version "1.3.1")
-- Could NOT find LZO
-- Found JPEG: C:/gtk-build/gtk/x64/release/lib/jpeg.lib (found version "62")
-- Performing Test FOUND_LIBJPEG_TURBO
-- Performing Test FOUND_LIBJPEG_TURBO - Success
-- Found PNG: C:/gtk-build/gtk/x64/release/lib/libpng16.lib (found version "1.6.44")
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
-- Could NOT find SDL2 (missing: SDL2_LIBRARY SDL2_INCLUDE_DIR)
-- Some or all of the gtk libraries were not found. (missing: GTK2_GTK_LIBRARY GTK2_GTK_INCLUDE_DIR GTK2_GDK_INCLUDE_DIR GTK2_GDKCONFIG_INCLUDE_DIR GTK2_GDK_LIBRARY GTK2_GLIB_INCLUDE_DIR GTK2_GLIBCONFIG_INCLUDE_DIR GTK2_GLIB_LIBRARY)
-- Could NOT find GnuTLS (missing: GNUTLS_LIBRARY GNUTLS_INCLUDE_DIR) (Required is at least version "3.4.0")
-- Found OpenSSL: C:/gtk-build/gtk/x64/release/lib/libcrypto.lib (found version "1.1.1g")
-- Could NOT find FFMPEG (missing: FFMPEG_INCLUDE_DIRS FFMPEG_LIBRARIES avformat avcodec avutil swscale) (Required is at least version "3.1.0")
...
...
...
[71/95] Building C object CMakeFiles\test_cargstest.dir\test\cargstest.c.obj
[72/95] Linking C executable examples\client\ppmtest.exe
[73/95] Building C object CMakeFiles\test_encodingstest.dir\test\encodingstest.c.obj
C:\gtk-build\build\x64\release\vncserver\test\encodingstest.c(97): warning C4018: '>=': signed 또는 unsigned가 일치하지 않습니다.
[74/95] Linking C executable examples\server\cursors.exe
[75/95] Linking C executable examples\server\camera.exe
[76/95] Linking C executable examples\server\backchannel.exe
[77/95] Linking C executable examples\server\pnmshow.exe
[78/95] Linking C executable examples\server\example.exe
[79/95] Linking C executable examples\server\colourmaptest.exe
[80/95] Linking C executable examples\server\fontsel.exe
[81/95] Linking C executable examples\server\vncev.exe
[82/95] Linking C executable examples\server\storepasswd.exe
[83/95] Linking C executable examples\server\pnmshow24.exe
[84/95] Linking C executable examples\server\repeater.exe
[85/95] Linking C executable examples\server\simple.exe
[86/95] Linking C executable examples\server\rotate.exe
[87/95] Linking C executable examples\server\simple15.exe
[88/95] Linking C executable examples\server\regiontest.exe
[89/95] Linking C executable examples\server\blooptest.exe
[90/95] Linking C executable test\tjbench.exe
[91/95] Linking C executable test\tjunittest.exe
[92/95] Linking C executable test\cargstest.exe
[93/95] Linking C executable test\copyrecttest.exe
[94/95] Linking C executable test\encodingstest.exe
[94/95] Install the project...
-- Install configuration: "RelWithDebInfo"
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/vncclient.lib
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/keysym.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/threading.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfb.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbclient.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbconfig.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbproto.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbregion.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/vncserver.lib
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/keysym.h
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/threading.h
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfb.h
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbclient.h
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbconfig.h
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbproto.h
-- Up-to-date: C:/gtk-build/build/x64/release/vncserver-rel/include/rfb/rfbregion.h
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/cmake/LibVNCServer/LibVNCServerTargets.cmake
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/cmake/LibVNCServer/LibVNCServerTargets-relwithdebinfo.cmake
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/cmake/LibVNCServer/LibVNCServerConfigVersion.cmake
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/cmake/LibVNCServer/LibVNCServerConfig.cmake
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/pkgconfig/libvncserver.pc
-- Installing: C:/gtk-build/build/x64/release/vncserver-rel/lib/pkgconfig/libvncclient.pc

Downloading packages
Building project cmake (3.31.2)
Building project ninja (1.12.1)
Building project nasm (2.13.03)
Building project libjpeg-turbo (2.0.4)
Building project zlib (1.3.1)
Building project libpng (1.6.44)
Building project perl (5.20.0)
Building project msys2 ()
Building project openssl (1.1.1g)
(git) Updating directory C:\gtk-build\src\git-exp\vncserver
Building project vncserver (git/master)

Project(s) built:
    cmake                     (0.001 s)
    ninja                     (0.001 s)
    nasm                      (0.001 s)
    libjpeg-turbo             (8.472 s)
    zlib                      (3.132 s)
    libpng                    (3.871 s)
    perl                      (0.001 s)
    msys2                     (0.001 s)
    openssl                   (210.775 s)
    vncserver                 (13.820 s)

```

--- a/gvsbuild/projects.py
+++ b/gvsbuild/projects.py
@@ -2113,11 +2113,11 @@ class Project_vncserver(GitRepo, CmakeProject):
 
     def build(self):
         CmakeProject.build(self,
-                           cmake_params=r'-DWITH_OPENSSL=ON'
-                                        r'-DWITH_GNUTLS=OFF'
-                                        r'-DWITH_GCRYPT=OFF'
-                                        r'-DWITH_ZLIB=ON'
-                                        r'-DWITH_JPEG=ON'
-                                        r'-DWITH_PNG=ON',
+                           cmake_params=('-DWITH_OPENSSL=ON'
+                                         '-DWITH_GNUTLS=OFF'
+                                         '-DWITH_GCRYPT=OFF'
+                                         '-DWITH_ZLIB=ON'
+                                         '-DWITH_JPEG=ON'
+                                         '-DWITH_PNG=ON'),
                            use_ninja=True,
                            out_of_source=False)

--- a/gvsbuild/projects.py
+++ b/gvsbuild/projects.py
@@ -341,9 +341,9 @@ class Gperf(GitRepo, Meson):
         Meson.__init__(
             self,
             "gperf",
-            repo_url="https://github.com/hsccr/gperf.git",
+            repo_url="https://github.com/rurban/gperf.git",
             fetch_submodules=False,
-            tag = None,
+            tag="c24359b4eab86d71c655c3b3fc969f13aac879ce",
             dependencies=["ninja"],
         )
 

